easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1027,9 +1043,12 @@ def robot_find_easyconfig(name, version):
             if os.path.isfile(easyconfig_path):
                 _log.debug("Found easyconfig file for name %s, version %s at %s" % (name, version, easyconfig_path))
                 _easyconfig_files_cache[key] = os.path.abspath(easyconfig_path)
-                return _easyconfig_files_cache[key]
+                res = _easyconfig_files_cache[key]
```

Comment:
mostly style, returning somewhere 'deep' in the function results in code that's harder to read or understand at a glance (it's not immediately clear if there are other `return`s for example)